### DISCUSSION TITLE
Add Epic Launcher and EA launcher to apps

### DISCUSSION THREAD

**Discussion by jackydany** [LizardByte/community#148] [Sunshine]
Hi there,
i try to add the 2 launchers to the apps to use them out of moonlight without having to use the desktop-stream first.
but if i use the epic launcher exe file in the apps path, the connection will always be canceled.

can someone tell me the correct way to add these two to the apps list?

Thanks a lot

**Comment by ReenigneArcher** [comment]
Add the command as a detached command. More info here: https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/usage.html#application-list

**Reply by jackydany** [reply]
i dont know what you mean by that, sorry.

**Reply by GoPadge** [reply]
And the link's dead.

**Reply by ReenigneArcher** [reply]
You don't need to use detached anymore as auto detach is the default.

https://docs.lizardbyte.dev/projects/sunshine/latest/md_docs_2getting__started.html#application-list

https://docs.lizardbyte.dev/projects/sunshine/latest/md_docs_2app__examples.html

**Comment by xoniq** [comment]
Anything on this?

I can for example start Tomb Raider by avoiding the Epic launcher to directly target the exe file, but for GTA V I cannot, it then cannot connect the Epic launcher which is active, because something isn't passed through.
